website/docs/topics/feature-flags/best-practices-using-feature-flags-at-scale.md

```diff
@@ -0,0 +1,350 @@
+
+### 12. Don’t use flags to manage configuration.
+
+Both feature flags and configuration settings control an application’s behavior. However, it's crucial to distinguish between feature flagging systems and configuration management systems and use each for its intended purpose.  Otherwise, you risk management nightmares and security vulnerabilities
+
+Feature flags are intended for dynamic and temporary use, giving you [runtime control](https://docs.getunleash.io/topics/feature-flags/runtime-control) to enable or disable functionality in a live environment. This flexibility supports [short-lived experiments](https://docs.getunleash.io/topics/feature-flags/short-lived-feature-flags), such as A/B testing or gradual rollouts, instead of relying on configuration settings that require an application redeploy to take effect. Despite the ease of turning off and on features or functionality with feature flags, they should not be repurposed for managing configuration.
+
+A dedicated config management system should manage long-term configuration settings. These systems are designed to handle sensitive information such as API keys and access credentials securely, unlike feature flag systems, which typically do not encrypt flag states. Misusing feature flags for configuration can lead to security risks and management complexities.
+
+A good rule of thumb is that if the data is static (you don’t expect it to change without restarting your application), needs encryption, or contains PII (Personal Identifiable Information), it’s probably better suited for your Configuration management system than a feature flag system.
+
+
+<table>
+  <tr>
+   <td>Configuration system
+   </td>
+   <td>Feature flag system
+   </td>
+  </tr>
+  <tr>
+   <td>Is: Long-lived, static during runtime
+   </td>
+   <td>Is: Short-lived, changes during runtime
+   </td>
+  </tr>
+  <tr>
+   <td>Examples of things to update with the config system: 
+<ul>
+
+<li>Database or server credentials
+
+<li>Server port
+
+<li>CORS headers
+
+<li>API base URL
+
+<li>SSO setup
+</li>
+</ul>
+   </td>
+   <td>Examples of things to update with feature flag system:
+<ul>
+
+<li>Dark mode/light mode
+
+<li>Stable / Beta feature
+
+<li>New or old algorithm
+
+<li>A/B test
+</li>
+</ul>
+   </td>
+  </tr>
+</table>
+
+### 13. Never reuse feature flag names.
+
+Feature flag names need to be globally unique.  [We wrote about this before](https://docs.getunleash.io/topics/feature-flags/unique-names), but it is such an important point that it bears repeating. Every feature flag within the same Feature Flag Control service must have a unique name across the cluster to prevent inconsistencies and errors.
```

Comment:
This isn't a good link title because it requires context to understand what it points to. Consider something like this instead. Also, whenever you link to other docs, you should link directly to the file, not to the URL, as explained in the [docusaurus markdown links](https://docusaurus.io/docs/markdown-features/links) article. In particular, this means the build will break if the doc doesn't exist anymore instead of the build succeeding but the link giving you a 404.

```suggestion
We've said it before, but we'll say it again: [Feature flag names need to be globally unique](feature-flags-best-practices.md#8-use-unique-names-across-all-applications-enforce-naming-conventions). Every feature flag within the same Feature Flag Control service must have a unique name across the cluster to prevent inconsistencies and errors.
```